src/material.cpp

```diff
@@ -843,7 +843,8 @@ void pyne::Material::from_text(std::string filename) {
     } else if (keystr == "APerM") {
       f >> valstr;
       atoms_per_molecule = pyne::to_dbl(valstr);
-    } else if (pyne::nucname::isnuclide(keystr)) {
+    } else if (pyne::nucname::isnuclide(keystr) ||
+               pyne::nucname::iselement(keystr)) {
```

Comment:
Thanks for keeping me honest, @katyhuff :). I have added a test and removed more whitespace.